.github/workflows/go.yml

```diff
@@ -62,3 +62,7 @@ jobs:
         with:
           name: junit.xml
           path: test_reports/junit.xml
+      - name: Display lint results
+        run: golangci-lint run
```

Comment:
Your CI job is failing because you don't have this tool installed before running this command.